alpha/apps/kaltura/lib/myPackagerUtils.class.php

```diff
@@ -456,7 +456,7 @@ public static function getPreferredStorageId($dcId)
 			$preferredStorageId = myCloudUtils::getCloudPreferredStorage();
 		}
 
-		if(!$preferredStorageId)
+		if(!$preferredStorageId && myCloudUtils::isEnvironmentWithCloudStorage())
```

Comment:
Why not use isClodDc?

Reply:
@yossipapi isCloudDc will return if the dc is on cloud not if there is a dc with cloud in the environment setup